src/wms/src/test/java/org/geoserver/wms/capabilities/GetCapabilitiesReponseTest.java

```diff
@@ -113,6 +113,24 @@ public void testInternationalContent() throws Exception {
                 "a i18n abstract for group nature", natureGroup + "/Abstract", result);
 
         assertXpathEvaluatesTo("english keyword", natureGroup + "/KeywordList/Keyword", result);
+
```

Comment:
same as before

Reply:
[added a new test and removed the previous modifications.](url) added a new test and removed the previous modifications.